backend/src/main/java/eu/clarin/switchboard/resources/ToolsResource.java

```diff
@@ -4,23 +4,31 @@
 import eu.clarin.switchboard.app.config.ToolConfig;
 import eu.clarin.switchboard.core.Tool;
 import eu.clarin.switchboard.core.ToolRegistry;
-import eu.clarin.switchboard.profiler.ProfileMatcher;
+import eu.clarin.switchboard.profiler.api.Confidence;
+import eu.clarin.switchboard.profiler.api.Profile;
 import org.slf4j.LoggerFactory;
 
 import javax.ws.rs.*;
-import javax.ws.rs.core.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URLConnection;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.List;
+import java.util.Map;
 
 @Path("")
 public class ToolsResource {
     private static final ch.qos.logback.classic.Logger LOGGER = (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(ToolsResource.class);
 
+    public static final String QUERY_PARAM_MEDIATYPE = "mediatype";
+    public static final String QUERY_PARAM_LANGUAGE = "language";
+    public static final String QUERY_PARAM_ONLY_PRODUCTION_TOOLS = "onlyProductionTools";
+
```

Comment:
I assume the intention was to use these instead of the values in the parameter annotations (as well as elsewhere in the code?).

Reply:
They're leftovers from an unsuccessful path.